### PR TITLE
[Task] Create and Deploy Serverless Artillery test for /Users/{UserID}/Tasks/{TaskID}

### PR DESCRIPTION
**Pull request by gillilands19**
**What Is it**
This pull request completes the performance test and sets up the test folder with the dependencies needed for slsart to run and deploy resources for slsart tests.

**Info**
Has a bug. AWS github user does not have correct permissions to fully deploy slsart resources and tests.

**Time Spent**
2 hours researching serverless artillery and following the lab
1.5 hours debugging deployment errors and reading source code
1 hour writing new test and deploying

### CODE REVIEW

**Review by StopMakingUsernamesUnique — APPROVED** [review]
We will probably need to continue debugging testing after merge, please review PRS in the slack channel if you haven't already

File: test/artillery/.slsart [hunk 1]
```diff
@@ -0,0 +1 @@
+version: 0.0.1
```
Comment (toddysm) [inline test/artillery/.slsart:1]: Do we need all other files in addition to the YAML?

File: test/artillery/analysis.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      return script.sampling.errorBudget
+    } else if (defaultErrorBudget !== undefined) {
+      return defaultErrorBudget
+    } else {
+      return sampling.defaults.sampling.errorBudget
+    }
+  },
+
+  /**
+   * Analyze a set of sampling reports, each of which is the result of a sample battery's execution
```
Comment (chonger878) [inline test/artillery/analysis.js:16]: I appreciate you adding documentation for the parameters and what each function does.  It really helps me figure out the logic of each line of code here! :)

File: test/artillery/artillery-acceptance.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const sampling = require('./sampling.js')
```
Comment (chonger878) [inline test/artillery/artillery-acceptance.js:1]: Maybe it's just me, but perhaps I could recommend that you can make a quick comment or something that summarizes what artillery-acceptance, artillery-performance, and artillery monitoring does?  I mean, I guess the titles are descriptive enough....maybe I am just being nitpicky.

**Review by chonger878 — APPROVED** [review]
I just made some slight recommendations. I might be wrong...

**Pull request merged**